src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const adder = (number) => {
```

Comment:
It's a good practice to explicitly check for `undefined` using the `typeof` operator to avoid possible confusion with falsy values.